Documentation/methods/watchdogstats.md

```diff
@@ -0,0 +1,22 @@
+# watchdogstats
+
+## Description
+
+Returns some statistics about Watchdog & bans.
+
+## Parameters
+- key
+
+## Example Watchdog-Stats
```

Comment:
It's without a dash.

Reply:
Hm? I'm a little confused.

The dash is there for formatting? It makes it a bullet-point?
![image](https://user-images.githubusercontent.com/26250962/31351211-20f29fd0-ad22-11e7-8d82-6d0c9dbf0c49.png)